OpenDreamClient/Interface/Prompts/ColorPrompt.cs

```diff
@@ -0,0 +1,36 @@
+﻿using OpenDreamShared.Dream.Procs;
+using System;
+using System.Windows.Controls;
+using System.Windows.Media;
+using Xceed.Wpf.Toolkit;
```

Comment:
Why are you using WPF? This is not cross-platform.

Reply:
I was under the impression the client was already not cross platform. What should be used instead?

Reply:
It's only not cross-platform because WebView2 hasn't been finished for Linux/Darwin. The last comment from Microsoft on this matter was that they intended to do so by the end of this year.

Reply:
Good to know. I tried making this work with Windows.Forms.ColorDialog but I'm getting weird hangs that I don't know how to debug, so I'm closing this.

Reply:
WPF is already used for everything UI-related.

Reply:
`Microsoft.Web.WebView2.Wpf` that is